source/hands_on/gail.rst

```diff
@@ -80,6 +82,40 @@ Extensions
++=====================+=================+=====================================================+==========================+======================+
+|                     |                 |                                                     |`config_link_l <https://  |                      |
+|                     |                 |                                                     |github.com/opendilab/     |                      |
+|                     |                 |                                                     |DI-engine/tree/main/dizoo/| DQN                  |
+|Lunarlander          |  200            |.. image:: images/benchmark/lunarlander_gail.png     |box2d/lunarlander/config/ |                      |
+|                     |                 |                                                     |lunarlander_dqn_gail_     |                      |
+|(LunarLander-v2)     |                 |                                                     |config.py>`_              |                      |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+|                     |                 |                                                     |`config_link_b <https://  |                      |
+|                     |                 |                                                     |github.com/opendilab/     |                      |
+|Bipedalwalker        |                 |                                                     |DI-engine/tree/main/dizoo/| SAC                  |
+|                     |  300            |.. image:: images/benchmark/bipedalwalker_gail.png   |box2d/bipedalwalker/      |                      |
+|(BipedalWalker-v3)   |                 |                                                     |config/bipedalwalker_sac_ |                      |
+|                     |                 |                                                     |gail_config.py>`_         |                      |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+|                     |                 |                                                     |`config_link_h <https://  |                      |
+|                     |                 |                                                     |github.com/opendilab/     |                      |
+|Hopper               |                 |                                                     |DI-engine/tree/main/dizoo/| SAC                  |
+|                     |  3500           |.. image:: images/benchmark/hopper_gail.png          |mujoco/config/hopper_sac_ |                      |
+|(Hopper-v2)          |                 |                                                     |gail_default_config.py>`_ |                      |
```

Comment:
Hopper-v2 or Hopper-v3?

Reply:
You are right, is v3, thank you for pointing it out.